### PR TITLE
Upgrade to Keycloak 18

### PR DESCRIPTION
**Pull request by MancunianSam**
- Pull the image from the new docker repo.
- Update all the folder names to remove jboss
- Update the realm import to call the new start script.
- Update the realm update script to use the new url.

This will need to be updated with the new `tdr-configurations` hash once https://github.com/nationalarchives/tdr-configurations/pull/108 has been merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ To run, build and test locally:
     * Run the following commands in the root directory:  `[root directory] $ npm install` and `[root directory] $ npm run build-theme`
         * this will compile the theme sass, copy the static assets to the theme `resource` directory and compile the typescript for WebAuthn.
 4. Build both spi jars:
-    * From the root directory run the following command: `sbt govUkNotifySpi/assembly eventPublisherSpi/assembly`
+    * From the root directory run the following command: `sbt assembly`
```
Comment (TomJKing) [inline README.md:182]: Not sure these instructions here: https://github.com/nationalarchives/tdr-auth-server/blob/master/README.md#update-tdr-theme-locally will still work now the `standalone-ha.xml` is gone?

I think might need to go through in the whole `README` to check it.

**Review by TomJKing — COMMENTED** [review]
Looks good. Just some minor comments.

File: README.md (hunk 1 above)
Comment (MancunianSam) [reply README.md:142]: Some of that README references stuff that hasn't been true for a year or so. I've gone through and tried to update it where it's wrong.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Review by TomJKing — APPROVED** [review]

**Pull request merged**